### PR TITLE
fixed a lot of grammar problems

### PR DESCRIPTION
**Pull request by zevav**
Capitalized all "Anaconda"s, fixed other grammar issues and awkward phrasing, and broke down a run-on sentence (starting with "when you are writing).  Made "I/we" consistent.

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -18,110 +18,108 @@ Platform | Status | Development | Maintainer
...
-system `PATH` but this behavior can be configured globally or per project (
+system `PATH`, but this behavior can be configured globally or per project (
 take a look at [Configuring Anaconda the Right Way](http://localhost:1313/anaconda/anaconda_settings/)
 for detailed information about that).
 
 In this way, we solve two common problems in Sublime Text plugins:
 
 * Plugins have to work/use the Python version of the embedded Sublime Text runtime (Python 3.3.3).
-* Plugins usually make Sublime Text 3 unresponsive (or completely freeze) when they do weight process or calculations even using Threads as the embedded Sublime Text Python interpreter also have a [GIL](https://wiki.python.org/moin/GlobalInterpreterLock) that get locked
+* Plugins usually make Sublime Text 3 unresponsive (or completely frozen) when they do weight process or calculations, even when using Threads, as the embedded Sublime Text Python interpreter also has a [GIL](https://wiki.python.org/moin/GlobalInterpreterLock) that gets locked.
```
Comment (zevav) [inline content/about.md]: I'm not sure about "weight process" -- do you mean "weighty processing" or "a weighty process", or something else?  As it is, this phrase is unclear.

Comment (DamnWidget) [inline content/about.md]: Hmmm, I meant process that could take a while to complete as they process too much information or their implementations are not that optimal and it's growing complexity time is not so good.

**Comment by DamnWidget** [comment]
Hi @zevav 

First of all let me thank you about your contribution, contributions are always appreciated but documentation ones much more :)

I am not native English speaker, in fact I didn't spoke any English at all three or four years ago so there are loads and loads of weird phrases or grammatical errors all around, really appreciate the effort.

Do you think you can rebase your commit into a single big one commit with all the changes?

Thanks!

**Comment by zevav** [comment]
My pleasure.  I absolutely can rebase it, if you can help me with some instructions?  I tried cloning my fork locally, but that doesn't contain my commits... Sorry, I'm still a little n00b-ish with git.

**Comment by DamnWidget** [comment]
Your commits are in the branch patch-1, anyway, rebasing is an advance git matter, I will do it no worries :)

**Comment by zevav** [comment]
thanks -- if there's any helpful blog posts about it, i'm eager to learn.  :-)

Zev Averbach
Averbach Transcription
(212) 392-0149
www.avtranscription.com



On Mon, Jan 12th, 2015 at 04:55 PM <notifications@github.com>  wrote :

**Comment by DamnWidget** [comment]
Hi again.

Here is the best literature that you could consume about Git rebasing -> http://git-scm.com/book/en/v2/Git-Branching-Rebasing

Rebasing is as powerful as dangerous, take care ;)

**Comment by DamnWidget** [comment]
I've merged this manually (to rebase it) so It's already in the master branch, I have to regenerate the http and update the website now.

Thanks.

**Pull request closed without merging**